src/HybridDb/StoreStats.cs

```diff
@@ -10,5 +12,56 @@ public class StoreStats
         public long NumberOfQueries { get; set; } = 0;
 
         public Guid LastWrittenEtag { get; set; }
+
+        public int NumberOfNumberUndisposedConnections => connections.Count;
+        public int NumberOfUndisposedTransactions => transactions.Count;
+
+        public static event EventHandler<OnConnectionCreatedEventArgs> OnConnectionCreated;

```

Comment:
Det her er vores tredje event "system" i Hybrid nu. Queue bruger rx, store har også events, som man registrerer som actions og nu også disse. Lad os lige vende om vi skal konsolidere noget af det

Reply:
Det var kun fordi du nævnte events. Jeg har ikke brug for dem til det jeg har lavet. Jeg fjerner dem igen.